code/player/TTTPlayer.cs

```diff
@@ -215,12 +212,15 @@ protected override void UseFail()
 
         public override void StartTouch(Entity other)
         {
-            if (_timeSinceDropped < 1)
+            if (IsClient)
             {
                 return;
             }
 
-            base.StartTouch(other);
+            if (other is PickupTrigger)
+            {
+                StartTouch(other.Parent);
+            }
```

Comment:
No base call anymore?
And no drop pickup prevention?

Reply:
```
		public override void StartTouch( Entity other )
		{
			if ( IsClient ) return;

			if ( other is PickupTrigger )
			{
				StartTouch( other.Parent );
				return;
			}

			Inventory?.Add( other, Inventory.Active == null );
		}
```

This is what base looks like, we do not want to do `Inventory?.Add( other, Inventory.Active == null );` anymore. Maybe for things that aren't `TTTWeapons`? Thoughts?

Reply:
Hmmm... Maybe if it comes to that point, but not for now... Ty for the explaination